test/unit/object.spec.js

```diff
@@ -1,4 +1,4 @@
-import MnObject from '../../src/object';
+import Object from '../../src/object';
```

Comment:
This works?

Reply:
Babel converts:
```js
import Object from '../../src/object';

new Object();
```
to:
```js
'use strict';

var _object = require('../../src/object');

var _object2 = _interopRequireDefault(_object);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

new _object2.default();
```

Reply:
so in light of this working for babel.. and working apparently in node..  do we rename it?

Reply:
But in export will work? In this case renaming is not possible

Reply:
actually exporting works as well.  we should really test this in node, but I think it's just _recommended_ not to export `Object` it looks like it will work.  I'm kind of thinking we would be fine renaming it back to `Object`

Reply:
Is asking for trouble

```
import Object from 'backbone.marionete'

export default Object.extend({
  initialize() {
    this.keys = Object.keys(this)
  }
})
```

Reply:
for now @wesmangum probably best to switch this to `MnObject`